### ISSUE TITLE
Unclear variable assignment

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#496]
In the preprocessing step, it says that `package_document_as_text` parameter contains the package document as a utf-8 string.

The first step then says:

> LET package_document_content be textual representation of the Package document for the publication given package_document_as_text. 

I don't follow what this point of this assignment is, though. It's just passing the string from one variable to another. Nothing happens until the next step when `package_document_content` is parsed into a dom tree.

Unless there's a purpose I'm missing, I'd suggest removing this step and just parsing the text string that's sent to the function.

**Comment by mattgarrish** [comment]
This also applies to the onix techniques.
